Lib/posixpath.py

```diff
@@ -412,11 +387,11 @@ def realpath(filename, *, strict=False):
     """Return the canonical path of the specified filename, eliminating any
 symbolic links encountered in the path."""
     filename = os.fspath(filename)
-    path, ok = _joinrealpath(filename[:0], filename, strict, {})
+    path, _ = _joinrealpath(filename[:0], filename, strict, {})
```

Comment:
Please revert this renaming, which will have no impact on how users will interact with this code, and so does not belong in this PR